python/src/aiconfig/editor/client/src/components/GlobalParametersContainer.tsx

```diff
@@ -37,6 +37,7 @@ export default memo(function GlobalParametersContainer({
               <ParametersRenderer
                 initialValue={initialValue}
                 onUpdateParameters={onUpdateParameters}
+                maxHeight="300px"
```

Comment:
Moving this here so we can remove the hard-coded default from ParametersRenderer